postgres_composite_types/__init__.py

```diff
@@ -317,6 +290,23 @@ def __eq__(self, other):
                 return False
         return True
 
+    @classmethod
+    def adapter(cls, value):
+        """
+        Adapter to format instances of this model as the correct Postgres
+        snippet, including casts, for queries.
+        """
+        # Make a string like (value1, value2)::db_type
+        values = [adapt(field.value_from_object(value)).getquoted()
+                  for _, field in cls._meta.fields]
+        # For some reason, adapt().getquoted() returns bytes, but AsIs requires
+        # a str. Using bytes in AsIs leads to a bad time. Formatting a series
+        # of bytes is also not fun
+        query = b''.join([
```

Comment:
I'm curious about tests for storing utf-8 in the database, this generally works. Also a test for the `ArrayField`.